Vault 2.0 UI known issues (#2213)
hashicorp/web-unified-docs: content/vault/global/partials/important-changes/known-issue/secrets-engines-items-per-page-hides-results.mdx

### Secrets Engines items-per-page change can hide results ((#secrets-engines-items-per-page-hides-results))

<!-- BEGIN: Vault:=v1.21.x -->

| Change      | Status | Vault edition | Affected version | Fixed version |
| ----------- | ------ | ------------- | ---------------- | ------------- |
| Known issue | Open   | All           | 1.21.0           | No            |

<!-- END: Vault:=v1.21.x -->

> 

Reviewer: ```suggestion

<!-- BEGIN: Vault:=v2.x -->
```

| Change      | Status | Vault edition | Affected version | Fixed version |
| ----------- | ------ | ------------- | ---------------- | ------------- |
| Known issue | Open   | All           | 2.0.0            | 2.0.1         |

> 

Reviewer: ```suggestion

<!-- END: Vault:=v2.x -->
```
Each version entry needs its own gate as ungated content appears across all versions

#### Issue

On the **Secrets Engines** page in the Vault UI, changing **Items per page**
from a later results page can leave the table empty even when the item count

> still indicates results are available.

Reviewer: ```suggestion
Changing the number of items per page on the **Secrets Engines** page in the
Vault GUI from anything other than the first page of results can leave the table
empty even when the item count indicates available results.
```

This can happen when users start on page 2 or later with a smaller page size,
such as `5`, and then increase the page size to `25`, `50`, or `100`. The issue

> is limited to the UI list view and does not affect the underlying mounts.

Reviewer: ```suggestion
For example, if a user starts on page 3 with 5 results per page, then increases
the page size to 25. The list view still only shows the 5 original results.

The empty display table is strictly a cosmetic issue and does not affect the
underlying mounts.
```
